edgexfoundry-holding/device-onvif-camera
# refactor!: Inconsistent naming of Edgex commands

BREAKING CHANGE: Making Edgex command names consistent by removing `GET` and `SET` keywords.

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>

## Testing Instructions
make test
make docker
/compose-builder$ git checkout main
/compose-builder$ make pull
/compose-builder$ make gen no-secty ds-onvif-camera
Replace onvif service docker image in docker-compose.yml with your latest image created using make docker from the previous step
/compose-builder$ make up
/bin/configure-subnets.sh without error
/bin/map-credentials.sh without error
Onvif devices should be registered in Edgex
Get the updated postman collection and env from the doc folder, update the env with the names of onvif cameras and run the collection without error.

## New Dependency Instructions (If applicable)
